core/server/data/exporter/table-lists.js

```diff
@@ -36,7 +36,8 @@ const BACKUP_TABLES = [
     'members_status_events',
     'members_paid_subscription_events',
     'members_subscribe_events',
-    'members_product_events'
+    'members_product_events',
+    'temp_member_analytic_events'
```

Comment:
I think it's fine to include this in the export - but I was originally planning to not include it - I couldn't actually find a way to do that though - cc @naz is it possible?

There doesn't seem to be an "EXCLUDE_TABLES" list, and the tests would need updating to be able to handle excluding it - but I wonder if the tests were written that way because we _never_ want to exlude tables?

Reply:
I don't think it's worth including it in the export if that's a temporary table. If there's no special need for it would avoid adding future maintenance headache.

As for `EXCLUDE_TABLES`, there's no such concept at the moment because we assume be default to exclude every table that's not included in the `BACKUP_TABLES`. I probably don't fully understand the purpose of the `EXCLUDE_TABLES`, can you help me understand why we'd need them?

Reply:
Yeah I was thinking that as well, and then I realised that this is for the backups that Pro does - I think although it's temporary it might be around for a while whilst we test.

@ErisDS Do you have any thoughts on whether or not this table should be included in the backup export?

> I probably don't fully understand the purpose of the EXCLUDE_TABLES, can you help me understand why we'd need them?

The tests are written in such a way that they expect every table to be exported in the backup, so we'd need an exclude list for the tests to ignore - that's all :)

Reply:
In these cases we should always work with allow lists, not block lists. The code should be refactored so the tests can work from the same allowlist IMO?

> Yeah I was thinking that as well, and then I realised that this is for the backups that Pro does

I don't follow! This is a temporary table, why would we need a backup? Also exports != backups 😬  backups are done with mysqldump. Exports are meant to be for content.

Reply:
Urgh somehow hit a keyboard combo to submit, but in short, this table def should not be added to the allow list IMO.

Reply:
👌 Awesome okay - I'll get this removed and refactor ze tests to work with it!